Scripts/ExtractDomainAndFQDNFromUrlAndEmail/ExtractDomainAndFQDNFromUrlAndEmail.py

```diff
@@ -0,0 +1,127 @@
+import demistomock as demisto
```

Comment:
this script is basically the the same as the old one for extract FQDN + some minor adjustments, right?

Reply:
Exactly